apps/chrome-devtools/src/extension/wrap.ts

```diff
@@ -13,33 +17,42 @@ window.postMessage({
   }
 } as OtterMessage<ConnectContentMessage, typeof applicationMessageTarget>, '*');
 
+declare namespace globalThis {
+  let localMessageListener: ((this: Window, ev: MessageEvent<any>) => any) | undefined;
+}
+
 // Remove previous listener (if the script is reloaded by a previous extension instance)
-if ((globalThis as any).localMessageListener) {
-  window.removeEventListener('message', (globalThis as any).localMessageListener);
+if (globalThis.localMessageListener) {
+  window.removeEventListener('message', globalThis.localMessageListener);
 }
 
 /**
  * Listener for messages from the page
  * @param event
  */
-// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
-(globalThis as any).localMessageListener = function (event: MessageEvent<any>) {
+
+// eslint-disable-next-line prefer-arrow/prefer-arrow-functions -- expecting fat function to handle the context correctly
```

Comment:
I don't see why we couldn't use arrow function here, we don't make use of the context

Reply:
I think `window.addEventListener` enforce fat function